pkg/kubelet/reason_cache.go

```diff
@@ -40,8 +40,8 @@ type ReasonCache struct {
 	cache *lru.Cache
 }
 
-// Reason is the cached item in ReasonCache
-type reasonItem struct {
+// ReasonItem is the cached item in ReasonCache
+type ReasonItem struct {
```

Comment:
Do we need to export this? Can we unexport ReasonCache instead?

Reply:
Good catch.

I forgot that ReasonCache is not in a separate subpackage. 
+1 for unexporting it.

Reply:
Thanks @liggitt @yujuhong 
But `NewReasonCache` wants an exported `ReasonCache`.
```
// NewReasonCache creates an instance of 'ReasonCache'.
func NewReasonCache() *ReasonCache {
	return &ReasonCache{cache: lru.New(maxReasonCacheEntries)}
}
```
They are very difficult to satisfy at the same time, so I won't modify it for now.

Reply:
Can you review this one again? @yujuhong